### PR TITLE
Office 2021Q1 report

### PR DESCRIPTION
**Pull request by lwhsu**

### CODE REVIEW

File: 2022q1/office.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Contact: FreeBSD Office team ML <office@FreeBSD.org> +
+Contact: Dima Panov <fluffy@FreeBSD.org> +
+Contact: Li-Wen Hsu <lwhsu@FreeBSD.org>
+
+The FreeBSD Office team works on a number of office-related software suites and tools such as OpenOffice and LibreOffice.
+
+Work during this quarter was focused on providing the latest stable release of LibreOffice suite and companion apps to all FreeBSD users.
+
+During the 2022Q1 period we pushed maintenance patches for LibreOffice 7.2 leaf to the quarterly branch and bring the latest, 7.3, releases
```
Comment (pauamma) [inline 2022q1/office.adoc:15]: s/bring/brought/

Also, "leaf"? Not sure what you mean by that.

File: 2022q1/office.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+and all companion libraries such as MDDS, libIxion and more to the porttree.
```
Comment (pauamma) [inline 2022q1/office.adoc:16]: s/porttree/ports tree/ (consistency with name on https://cgit.freebsd.org/)

File: 2022q1/office.adoc [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Also we are still working on the link:https://github.com/fluffykhv/freebsd-ports-boost[Boost WIP repository] to bring the latest Boost library to the ports.
+
+We are looking for people to help with the open tasks:
+
+* The link:https://bugs.freebsd.org/bugzilla/buglist.cgi?bug_status=open&email1=office%40FreeBSD.org&emailassigned_to1=1&emailcc1=1&emailreporter1=1&emailtype1=substring&query_format=advanced&list_id=374316[open bugs list] contains all filed issues which need some attention
+* Upstream link:https://cgit.freebsd.org/ports/tree/editors/libreoffice/files[local patches in ports]
+
+Patches, comments and objections are always welcome in the mailing list and bugzilla.
```
Comment (pauamma) [inline 2022q1/office.adoc:25]: s/bugzilla/Bugzilla/

**Review by pauamma — CHANGES_REQUESTED** [review]

File: 2022q1/office.adoc (hunk 1 above)
Comment (lwhsu) [reply 2022q1/office.adoc:15]: @fluffykhv: maybe just call it "Libreoffice 7.2 port" ?

Comment (fluffykhv) [reply 2022q1/office.adoc:15]: Ah, yes. Sorry for misunderstood guys, last week was hard IRL, my mind is blowing :(

**Pull request merged**